ecommerce/enterprise/tests/mixins.py

```diff
@@ -564,24 +563,29 @@ def mock_enterprise_catalog_api(self, enterprise_customer_uuid, raise_exception=
         Helper function to register the enterprise catalog API endpoint.
         """
         enterprise_catalog_api_response = {
-            'count': 10,
-            'num_pages': 3,
-            'current_page': 2,
+            'count': 2,
+            'next': urljoin(self.ENTERPRISE_CATALOG_URL, f'?enterprise_customer={enterprise_customer_uuid}/'),
```

Comment:
If this is meant to imitate page 2 of the response, I think the the next url would be different.
```suggestion
            'next': urljoin(self.ENTERPRISE_CATALOG_URL, f'?page=3&page_size=2'),
```

Reply:
`enterprise_customer` query shouldn't be included? 
I did not simulate 2 page of the answer but changed response

Reply:
Hm, do you mean that the new `enterprise-catalog` [endpoint](https://github.com/openedx/enterprise-catalog/blob/b6f0133fcffe87d2d68f2845f2b9a4732bc98573/enterprise_catalog/apps/api/v1/views/enterprise_catalog_crud.py#L22) should not accept a query parameter `enterprise_customer`? Because this implementation I moved to the new endpoint

pr to enterprise-catalog: https://github.com/openedx/enterprise-catalog/pull/468/files

Reply:
Ah, gotcha, hadn't gotten to that PR yet, makes sense.  But you still need paging information for the next URL right? If the endpoint is meant to provide more results on a second page, the URL for that should be consistent. It sounds like it would include the `enterprise_customer` but also the next page number and page size.